unit_testing/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "unit_testing",
+  "version": "1.0.0",
+  "description": "Unit testing of coding train website.",
+  "main": "coding-challange-yaml.js",
+  "scripts": {
+    "test": "jest"
+  },
+  "author": "MeaningOf42",
+  "license": "MIT",
```

Comment:
@shiffman Just want to confirm this is OK with you

Reply:
Oh sorry, didn't notice that. If it's not ok then I can change it. If I did that would there be a way of adding myself as a contributor? Also if this is problematic you may want to check the first comment of unit_testing/coding-challange-yaml.test.js

Reply:
Sorry, to clarify I was more referring to the licensing - it should be fine, but thought I'd defer anyway. I don't have an issue having your authorship in here, and celebrating contributors and their work is very important. Having said that, we should look into a standardised way of doing that at some point.

Reply:
MIT license is good by me! This whole repo should be MIT as far as I'm concerned. In terms of authorship I'm very happy for @MeaningOf42 to be listed in `package.json` as the original creator of the tests.  The good news is that github retains a history of contributions and that should probably be the primary document. As others contribute to the tests we could consider changing author to "The Coding Train Community" and link to https://github.com/CodingTrain/website/graphs/contributors. I'm open to your suggestions!

Reply:
Changing author to "The Coding Train Community" once more people have contributed sounds like a good ide.